src/components/RenderRouter/ContentItem.scss

```diff
@@ -281,7 +281,7 @@
         padding: 40vw 6vw 10vw;
         position: relative;
         width: 100vw;
-        height: 145vw;
+        //height: 145vw;
```

Comment:
On some of the other pages the text can get longer...so this should make it scale...